packages/common/src/Adaptor.js

```diff
@@ -67,6 +67,27 @@ export function fn(func) {
   };
 }
 
+/**
+ * A custom step (or operation) that will only execute if the condition returns a truthy value
```

Comment:
step is confusing because it means something else in lightning.

I've just seen you've taken this from `fn`, so lets tweak `fn`'s docs too!

Let's say:
```
Execute a block of arbitrary JavaScript code. Take state as an argument and MUST return state at the end
```